code/modules/vending/_vending.dm

```diff
@@ -851,8 +885,8 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 	switch(action)
 		if("vend")
 			. = vend(params)
-		if("select_colors")
-			. = select_colors(params)
+		//if("select_colors")
```

Comment:
I made all ui references or stuff touching items commented. 

Bacon told me it would still be possible to use so now I have made it so it only exists in the typescript file and isolated procs.